src/main/scala/org/ergoplatform/local/MempoolAuditor.scala

```diff
@@ -78,6 +85,18 @@ class MempoolAuditor(nodeViewHolderRef: ActorRef,
     context become working // ignore other triggers until work is done
   }
 
+  private def broadcastTransactions(): Unit = {
+    poolReaderOpt.foreach { pr =>
+      val txs = pr.digest(settings.nodeSettings.reBroadcastVolume)
```

Comment:
Let's give `digest` more descriptive name, such that makes the logic of this line readable.